src/components/sidebar/lineView/lineHeaderTable.scss

```diff
@@ -1,44 +1,82 @@
 @import '../../shared/styles/common';
 @import '../../shared/styles/form';
 
+$cellPadding: 5px 2px 5px 2px;
+
 .lineHeaderTableView {
     width: 100%;
+    padding-top: 20px;
 
     .lineHeaderTable {
         width: 100%;
-        font-size: $smallFontSize;
-        border-collapse: separate;
-        border-spacing: 0 5px; /* Padding top/bottom between table rows */
+        border-collapse: collapse;
 
-        td {
-            padding-left: 10px;
-        }
-        td:first-of-type {
-            padding: 0px;
+        .header {
+            display: flex;
         }
+
         .columnHeader {
             white-space: nowrap;
             text-align: start;
-            padding-left: 10px;
+            padding-left: $cellPadding;
         }
-        .columnHeader:first-of-type {
+
+        .columnHeader:first-child {
             padding-left: 0px;
         }
-        .timestampRow {
-            white-space: nowrap;
+
+        .lineHeaderTableRow {
+            border-bottom: 1px solid #dfdfdf;
+
+            .lineHeaderTableCalendarCell {
+                min-width: 125px;
+                padding: $cellPadding;
+                vertical-align: top;
+
+                .timeInput {
+                    padding-top: 0px;
+                }
+            }
+
+            .lineHeaderTableButtonCell {
+                vertical-align: top;
+                padding-top: $cellPadding;
+            }
         }
 
         .lineHeaderRowHighlight {
             background-color: $lightblue;
         }
-    }
 
-    .editLineHeaderButton {
-        font-size: 1.8em;
-        padding: 3px;
+        .lineHeaderButton {
+            display: flex;
+            font-size: $largeFontSize;
+            padding: 5px;
+            margin: 4px;
+            margin-bottom: auto;
+        }
+
+        .removeLineHeaderButton {
```

Comment:
I would have the remove button blue but keep the background color as it is now (white icon and red background when hovering)